slack-api-client/src/main/java/com/slack/api/methods/SlackApiResponse.java

```diff
@@ -1,4 +1,17 @@
 package com.slack.api.methods;
 
+import java.util.List;
+import java.util.Map;
+
 public interface SlackApiResponse {
+
+    /**
+     * Returns all the HTTP response headers in the API response. The keys are lower-cased.
+     */
+    Map<String, List<String>> getHttpResponseHeaders();
```

Comment:
To support the situation where a header appears in a response several times, I use `List<String>` for the values. I'm sure that this is a common design.

Reply:
Interesting, I don't know if I've seen this approach before, but your approach makes sense to me.
Out of curiosity, are there situations the Slack backend returns multiple values for a single header?
Perhaps we can add a test to ensure multiple values for a single header are represented properly in this construct?

Reply:
>Out of curiosity, are there situations the Slack backend returns multiple values for a single header?

No, there aren't at this moment but it may be in the future.

>Perhaps we can add a test to ensure multiple values for a single header are represented properly in this construct?

The okhttp library does this and we can trust its behavior but it'd be good to add some tests on our end.